osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -58,6 +61,11 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
             BeatmapSetInfo beatmapSetInfo = beatmap.BeatmapSetInfo;
             BeatmapInfo beatmapInfo = beatmap.BeatmapInfo;
 
+            string bpm = getBPMRange(beatmap);
+            string length = TimeSpan.FromMilliseconds((beatmap.Beatmap.HitObjects.Last().EndTime - beatmap.Beatmap.HitObjects.First().StartTime)).ToString(@"m\:ss");
+            string hitCircles = "" + beatmap.Beatmap.HitObjects.Count(h => h.GetType().ToString().Equals("osu.Game.Modes.Osu.Objects.HitCircle"));

```

Comment:
is HitCircle can't be used since the class is in another project and the project can't be added as reference because of circular dependency